cypress_test/integration_tests/desktop/writer/editable_area_spec.js

```diff
@@ -211,12 +211,14 @@ describe(['taga11yenabled'], 'Editable area - Basic typing and caret moving', fu
 		ceHelper.type('{enter}');
 		ceHelper.type('green red');
 		// Need to wait here after typing paragraph in
-		// order for caret position to stick later
-		cy.wait(200);
+		// order for caret position to stick later.
+		// FIXME: We shouldn't need to wait.
+		cy.wait(1000);
 		ceHelper.checkPlainContent('green red');
 		ceHelper.moveCaret('left', '', 4);
 		ceHelper.checkCaretPosition(5);
 		helper.getBlinkingCursorPosition('P2');
+		cy.wait(800);
```

Comment:
Not happy about the size of these waits, but these seemed to be the minimum values to get this test to pass consistently locally. Both of these are masking a real bug, I can easily replicate the failure with manual inputs and it doesn't mirror the behaviour in desktop Writer. This seems very similar to #10585 and I wouldn't be surprised if the underlying reason is basically the same.